hack/local-up-cluster.sh

```diff
@@ -823,12 +825,17 @@ EOF
       --v=${LOG_LEVEL} 2>&1 &
     PROXY_PID=$!
 
+    cat <<EOF > /tmp/kube-scheduler.yaml
+apiVersion: componentconfig/v1alpha1
+kind: KubeSchedulerConfiguration
+clientConnection:
+  kubeconfig: ${CERT_DIR}/scheduler.kubeconfig
+featureGates: ${FEATURE_GATES}
+EOF
     SCHEDULER_LOG=${LOG_DIR}/kube-scheduler.log
     ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" scheduler \
       --v=${LOG_LEVEL} \
-      --kubeconfig "$CERT_DIR"/scheduler.kubeconfig \
-      --feature-gates="${FEATURE_GATES}" \
-      --master="https://${API_HOST}:${API_SECURE_PORT}" >"${SCHEDULER_LOG}" 2>&1 &
```

Comment:
We don't need to provide information about master anymore? I just need to ensure that it's not a bug and was removed intentionally.

Reply:
the master config was in the config yaml, it's removed intentionally.